.ci/generate_bear_requirements.py

```diff
@@ -126,8 +126,9 @@ def write_gem_requirements(requirements):
     for requirement in requirements:
         gem_dependencies.append(
             {'name': requirement.package,
-             'version': requirement.version,
-             'required': requirement.require})
```

Comment:
I'm still uncertain about this. This flag is recording useful information, just not very useful for our purposes.
So i think we keep the flag, in the Requirement.
I'd like to check if removing it from Gemfile improves performance at all.